Sources/SwiftUIHosting/SwiftUIHostingView.swift

```diff
@@ -15,6 +15,7 @@ open class SwiftUIHostingView: UIView {
     case compressedSizeThatFits
   }
 
+  @MainActor
```

Comment:
Since ViewModifier requires MainActor, Configuration is isolated to MainActor to have baseModifier property.